### ISSUE TITLE
OpenXR-SDK-Source-release-1.0.13.tar.gz is missing

### ISSUE THREAD

**Issue by svenstaro** [KhronosGroup/OpenXR-SDK-Source#235]
Release 1.0.13 has the .asc file but not the actual payload. Could you upload that?

**Comment by rpavlik** [comment]
We're just using the one that GitHub generates - here's the debian watchfile though it's a little cryptic: https://salsa.debian.org/openxr-team/openxr-sdk-source/-/blob/debian/sid/debian/watch The 1.0.14 tarball is https://github.com/KhronosGroup/OpenXR-SDK-Source/archive/release-1.0.14.tar.gz - just copied the link from the releases page.

Would it help if I downloaded and then re-uploaded the tarball? I'm already downloading it to diff it against the source tree and sign it. (Edit: I just did that for the 1.0.14 release. if it's substantially easier, I'm fine with adding it to the procedure.)
